population/create_glpi_computer_redfish.py

```diff
@@ -1264,6 +1303,41 @@ def check_and_post_processor_item(
         return
 
 
+def parse_list(args: argparse.Namespace, machines: list):
+    """Reads machine information from the provided list and CLI args and runs the
+    relevant command
+
+    Args:
+        args (argparse.Namespace): Arguments passed in by the user via the CLI
+    """
+    print("Parsing machine file\n")
+    machine_list = ""
+    try:
+        f = open(args.machine_list, "r")
+        machine_list = f.readlines()
+        f.close()
+    except FileNotFoundError:
+        sys.exit("can't open %s" % (machine_list))
+
+    for line in machine_list:
+        if line[0] != "#":
```

Comment:
Chance to consider if comments should be more explicitly stripped, for instance from the end of lines as well

Reply:
I'm thinking of converting the format for `machine_list` to yaml, since we are using it elsewhere and it has better comment support. Comments aren't supported by python's csv module: https://bugs.python.org/issue1225769

Reply:
Yeah that might work well

Reply:
Will do in a separate PR